doc/main.html

```diff
@@ -200,13 +255,17 @@ <h4>introduction section</h4>
 
         <p>The <code>id</code> attribute shall be present and equal to <code>sec-introduction</code>.</p>
 
+        <p>If no Introduction is required, the section shall not be present.</p>
```

Comment:
```suggestion
        <p>The absence of a section indicates that the author did not provide introductory text.</p>
```

I am not sure we should prohibit an empty section.

Reply:
Wasn't sure how the validation and build worked, noted.